ospf6d/ospf6_asbr.c

```diff
@@ -1374,25 +1377,52 @@ static void ospf6_redistribute_show_config(struct vty *vty, struct ospf6 *ospf6)
 		nroute[info->type]++;
 		total++;
 	}
+	if (use_json) {
```

Comment:
Let's move the if(use_json) to the interior of the for loop.  I see absolutely no reason to have 2 separate for loops.  Just complicates code and leads to problems in the future when the command is updated and one side is forgotten.

Reply:
resolved